tests/Api/Response/ExampleMulticolloPostShipmentsResponse.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Tests\Api\Response;
+
+class ExampleMulticolloPostShipmentsResponse extends ExampleJsonResponse
+{
+    /**
+     * @return array[]
+     */
+    protected function getDefaultResponseContent(): array
+    {
+        return [
+            ['id' => '123'],
```

Comment:
Multicollo post doet hetzelfde wat een normale shipment post doet, je krijgt een `id` terug en een `reference_identifier`. (Onze post shipments response klopt dus niet eens)

Je kunt dus `ExamplePostShipmentsResponse` gebruiken. Als je nog wil dat er andere ids uitkomen oid kan je dat meegeven in de constructor.